### PR TITLE
Prepare for 0.25 release

### PR DESCRIPTION
**Pull request by cijothomas**
Preparing for the release (Sorry for the delays!).
The version is now unified as discussed in https://github.com/open-telemetry/opentelemetry-rust/issues/2084, so this is "0.25.0" release of OpenTelemetry Core Components.

### CODE REVIEW

File: opentelemetry-zipkin/Cargo.toml
Comment (lalitb) [inline opentelemetry-zipkin/Cargo.toml:33]: this should use 0.25, along with for semantic-convention ?

Comment (cijothomas) [reply opentelemetry-zipkin/Cargo.toml]: Good catch!!
For next release onwards, I really need to automate this.

**Review by lalitb — APPROVED** [review]
LGTM with proto dev fixes, Thanks for initiating the release :)

**Comment by cijothomas** [comment]
Will wait for at-least one more review from maintainers/approvers before merging.

**Review by utpilla — APPROVED** [review]

**Review by TommyCpp — APPROVED** [review]

**Comment by TommyCpp** [comment]
close #2084 
close #1966

**Pull request merged**
